openquake/qa_tests_data/classical/case_18/expected/uhs-rlz-1.csv

```diff
@@ -0,0 +1,5 @@
+# source_model_tree_path=('b1',),gsim_tree_path=('b12',),investigation_time=1.0
```

Comment:
It looks like we have an inconsistency in the format of the comment line that contains parameters. Here parameters are separated by `","` but somewhere else we have parameters separated by `", "`
Please check about this.